news/_posts/2022-12-23-playtest-20221223.md

```diff
@@ -0,0 +1,113 @@
+---
+kind: article
+title: "Playtest 20221223"
+author: "PunkPun"
+created_at: 2022-12-22 23:00 +0000
+disqus_id: "playtest-20221223"
+permalink: "/news/playtest-20221223/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Fixed a few pathfinder crashes
```

Comment:
```suggestion
* Fixed several pathfinder-related crashes
```
? (To not have "Fixed a few" repeated twice in a row, unless that is intentional.)